public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -136,16 +139,21 @@ export const uuidRx = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a
 
 export const isUuid = (id) => !!id.match(uuidRx);
 
-export const updatePanel = (panel: CustomPanelType) => async (dispatch, getState) => {
+export const updatePanel = (panel: CustomPanelType, onSuccess: string, onFailure: string) => async (dispatch, getState) => {
   try {
     if (isUuid(panel.id)) await updateSavedObjectPanel(panel);
     else await updateLegacyPanel(panel);
-
+    if (onSuccess) {
+      setToast(onSuccess)
+    }
     dispatch(setPanel(panel));
     const panelList = getState().customPanel.panelList.map((p) => (p.id === panel.id ? panel : p));
     dispatch(setPanelList(panelList));
-  } catch (err) {
-    console.log('Error updating Dashboard', { err, panel });
+  } catch (e) {    
```

Comment:
I'm about to believe that this try..catch block... especially the catch{} could be put in a wrapper function.
Maybe I'm over-engineering.

Reply:
I think its over engineering for now... My 2 cents is to push it into another PR if necessary. I think this PR is already doing a lot regarding the framework around how we want to handle success and error in redux world